.github/workflows/ios-test.yml

```diff
@@ -27,8 +27,17 @@ jobs:
                   node-version: 16
                   cache: "npm"
                   cache-dependency-path: "**/package-lock.json"
+            - name: Cache multiple paths
+              uses: actions/cache@v2
+              id: cache
+              with:
+                  path: "**/node_modules"
+                  key: ${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}
             - name: Lock xcode version
               run: cd platform/ios; make lockxcodeversion
+            - name: Install modules if needed
+              if: steps.cache.outputs.cache-hit != 'true'
+              run: ./build.js root-install && ./build.js install
```

Comment:
I find it a bit error prone to have an explicit dependency installation step, so far all you gotta do is run `build.js` and it works... Couldn't we add an option like `--skip-deps` or something?